lib/random_alpha_numeric.dart

```diff
@@ -0,0 +1,44 @@
+import 'dart:math';
+
+const _numericStart = 48;
+const _numericEnd = 57;
+const _lowerAlphaStart = 97;
+const _lowerAlphaEnd = 122;
+const _upperAlphaStart = 65;
+const _upperAlphaEnd = 90;
+
+Random _random = Random();
+
+/// Generates a random string of [length] with alpha-numeric characters.
+String randomAlphaNumeric(int length) {
```

Comment:
I think this could be simplified:

```
const _chars = 'AaBbCcDdEeFfGgHhIiJjKkLlMmNnOoPpQqRrSsTtUuVvWwXxYyZz1234567890';
Random _random = Random();
String randomAlphaNumeric(int length) => String.fromCharCodes(Iterable.generate(
    length, (_) => _chars.codeUnitAt(_random.nextInt(_chars.length))));
```

Reply:
Good point 👍